fe/src/main/java/org/apache/doris/common/DuplicatedRequestException.java

```diff
@@ -20,12 +20,24 @@
 public class DuplicatedRequestException extends DdlException {
```

Comment:
Could you please add some commet for this class. What is the difference between it and LabelAlreadyUsedException.
And what does a `request` here mean?

Reply:
Added